Source/Core/Core/Debugger/PPCDebugInterface.cpp

```diff
@@ -16,6 +17,33 @@
 #include "Core/PowerPC/PPCSymbolDB.h"
 #include "Core/PowerPC/PowerPC.h"
 
+void PPCPatches::Patch(std::size_t index)
+{
+  auto& patch = m_patches[index];
+  if (patch.value.empty())
+    return;
+
+  const u32 address = patch.address;
+  const std::size_t size = patch.value.size();
+  if (!PowerPC::HostIsRAMAddress(address))
+    return;
+
+  for (u32 offset = 0; offset < size; offset += 1)
```

Comment:
`offset++` (or `++offset`, either works) is generally how this is expressed elsewhere in the codebase.